phpBB/config/installer/container/services_install_database.yml

```diff
@@ -1,16 +1,36 @@
 services:
-    installer.install_database.create_schema:
-        class: phpbb\install\module\install_database\task\create_schema
+    installer.install_database.create_schema_file:
+        class: phpbb\install\module\install_database\task\create_schema_file
         arguments:
             - '@installer.helper.config'
             - '@installer.helper.database'
             - '@filesystem'
-            - '@installer.helper.iohandler'
             - '%core.root_path%'
             - '%core.php_ext%'
         tags:
             - { name: install_database_install, order: 10 }
 
+    installer.install_database.set_up_database:
+        class: phpbb\install\module\install_database\task\set_up_database
+        arguments:
+            - '@installer.helper.config'
+            - '@installer.helper.database'
+            - '@filesystem'
+            - '@installer.helper.iohandler'
+            - '%core.root_path%'
+        tags:
+            - { name: install_database_install, order: 13 }
```

Comment:
I'd prefer to have at least a step of 5 and not uneven steps (in this case 3 and 4).